DESCRIPTION

```diff
@@ -27,7 +27,8 @@ Suggests:
     pkgdown,
     rmarkdown,
     testthat,
-    tibble
+    tibble,
+    bit64
```

Comment:
Can we really just add `bit64` as suggest and not import? In `arrow`, it was easy to hit crashes if `bit64` gets unloaded, see https://github.com/apache/arrow/pull/2867, but maybe that's not the case in `vctrs`, just passing along FWIW.

Reply:
I think it's fine to use `Suggests` and then `bit64::` when we need a function from it.